src/api-reference/invoice/purchase-request-endpoints.markdown

```diff
@@ -0,0 +1,250 @@
+---
+title: Purchase Request
+layout: reference
+---
+
+# Purchase Request
+* [Create a new purchase request](#post)
+* [Get details of purchase request](#get)
+* [Schema](#create_purchase_request_schema)
+* [Response schema](#create_purchase_request_schema-response)
+* [Error codes](#error-codes)
+
+
+## Version
+4.0<br>
+**PRE-RELEASE:** The API is in Pre-Release and that only authorized early adopters should be using the API. Please contact <a href="mailto:charles.muellner@sap.com">charles.muellner@sap.com</a> for more information.
+
+## <a name="post"></a>Create a new purchase request  
+
+    POST  /purchaserequest/v1/purchaserequests
+
+Create a Purchase Request based on provided header and line item details. If the request is valid it returns back a unique identifier to look at purchase request details and creates a purchase request in back ground.
+
+### <a name="8"></a>Parameters  
+
+|Name | Type | Format | Description
+|-----|------|--------|------------
+|`X_AUTH_TOKEN`|`string`|`header`|**Required**: Bearer Token that identifies the caller. This is the company JWT
```

Comment:
Can we rename `X_AUTH_TOKEN` to `Authorization`

Reply:
Fixed in f847a0bc